### PR TITLE
Rpow

### PR DESCRIPTION
**Pull request by mhhf**
RRRRRrrrrrrrrrrr POW

### CODE REVIEW

File: lib/build.js [hunk 1]
```diff
@@ -366,7 +373,7 @@ const buildAct = config => ({act, oog, pass, name, hasGas, gas, gas_raw}) => {
       "k": "#execute ~> CONTINUATION => " + ((act.internal && pass && !(oog)) ? "#execute ~> CONTINUATION" : "#halt ~> CONTINUATION"),
     "ethereum.evm.callState.programBytes": `#parseByteStack("0x${config.contracts[act.subject].bin_runtime}")`,
     "ethereum.evm.callState.program":  `#asMapOpCodes(#dasmOpCodes(#parseByteStack("0x${config.contracts[act.subject].bin_runtime}"), PETERSBURG))`,
-    "ethereum.evm.callState.callData": act.callData + ` => _`,
+    "ethereum.evm.callState.callData": (act.callData || '_') + ` => _`,
```
Comment (MrChico) [inline lib/build.js]: act.callData is already set to `_` for internal specs

Comment (mhhf) [reply lib/build.js]: the new fined "lemma" thing is not internal and somehow it was undefined here so i had to put that here

Comment (MrChico) [reply lib/build.js]: Really? This line suggests otherwise https://github.com/dapphub/klab/pull/182/files#diff-26b681bd4dac10f25d5293c7595c435bR159

Comment (mhhf) [reply lib/build.js]: But it misses crucial stuff like an interface where the calldata is build from, it doesn't have all the information an internal function has

**Comment by mhhf** [comment]
I'll go ahead and merge this as well

**Pull request merged**